test.js

```diff
@@ -53,3 +57,6 @@ test('with -n, output has NO trailing newline', macro,
 test('with --no-newline, output has NO trailing newline', macro,
 	{args: ['--no-newline', 'red', 'bold', 'unicorn'], opts: {stripFinalNewline: false}},
 	chalk.red.bold('unicorn') /* No trailing newline */);
+
+test('demo', macro, {args: ['--demo']},
```

Comment:
I think this would be better as a snapshot test. See `t.snapshot` in AVA docs.

Reply:
Cool, thanks! Done in 7608084.